CMakeLists.txt

```diff
@@ -76,7 +76,9 @@ set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS 1)
 set (LIBHPDF_STDC_HEADERS 1)
 
 # support all of the different variations of LIBPNG defines in HARU
-set (LIBHPDF_HAVE_LIBPNG ${PNG_FOUND})
+if (${PNG_FOUND})
```

Comment:
My apologies, I was not very clear. The outer `if` statement can be removed. Similar to `LIBHPDF_HAVE_ZLIB` is defined below.

Reply:
It is strange, but it works. I treated `#ifdef LIBHPDF_HAVE_LIBPNG` as if it is _defined_, with no meaning to value, while `#if` checks value.

Well, seems I was wrong. Removed.